lib/ansible/plugins/filter/human_to_bytes.yml

```diff
@@ -4,10 +4,11 @@ DOCUMENTATION:
   short_description: Get bytes from string
   description:
     - Convert a human-readable byte or bit string into a number bytes.
+    - Will always use base 2 (1024) for the unit prefix, e.g 1 KB is 1024, 1MB is 1048576, etc.
```

Comment:
this is incorrect,  see when "b" is added or `isbits` option is used. At best you should say ' it defaults to the isbits value if no identifier is present'

Reply:
It might be a bug, but `12b` will parse to `12` when `isbits=false`. Any unit prefixes, e.g.  `12kb` will then throw an error: `'human_to_bytes() can''t interpret following string: 12kb'`.

With `isbits=true`, `12kb` will still evaluate to `13312`, so it's still correctly uses 1024. No conversion between bytes and bits happens in that code.

So I guess the line above is incorrect, though, as it *doesn't* convert a bit string to a number bytes.